docs/en/integrations/data-visualization/powerbi-and-clickhouse.md

```diff
@@ -4,107 +4,133 @@ slug: /en/integrations/powerbi
 ---
-
 import ConnectionDetails from '@site/docs/en/_snippets/_gather_your_details_http.mdx';
 
 # Power BI
 
-Power BI can load the data from ClickHouse Cloud or on-premise deployment using either
-the [ODBC driver](https://github.com/ClickHouse/clickhouse-odbc)
-or [ClickHouse Native connector](https://github.com/ClickHouse/power-bi-clickhouse). Both methods support Load mode, but
-the latter also supports Direct Query mode which eliminates the necessity to load the entire table.
+Microsoft Power BI can query or load into memory data from [ClickHouse Cloud](https://clickhouse.com/cloud) or a self-managed deployment.
 
-This tutorial will guide you through the process of loading data using either of these methods.
-<br/>
-<br/>
-<br/>
+There are several flavours of Power BI that you can use to visualise your data:
 
-# ClickHouse Native Connector
+* Power BI Desktop: A Windows desktop application for creating Dashboards and Visualisations
+* Power BI Service: Available within Azure as a SaaS to host the Dashboards created on Power BI Desktop 
 
-## 1. Gather your connection details
+Power BI requires you to create your dashboards within the Desktop version and publish them to Power BI Service. 
 
-<ConnectionDetails />
+This tutorial will guide you through the process of:
 
-## 2. Install ClickHouse ODBC Client
+* [Installing the ClickHouse ODBC Driver](#install-the-odbc-driver)
+* [Installing the ClickHouse Power BI Connector into Power BI Desktop](#install-clickhouse-connector)
+* [Querying data from ClickHouse for visualistion in Power BI Desktop](#query-and-visualise-data)
+* [Setting up an on-premise data gateway for Power BI Service](#power-bi-service)
 
-Download the most recent ClickHouse ODBC release from [here](https://github.com/ClickHouse/clickhouse-odbc/releases).
-Execute the supplied `.msi` installer and follow the wizard.
-Optional "debug symbols" are not required, so you could keep everything default.
+## Prerequisites
 
-<img src={require('./images/powerbi_01.png').default} class="image" alt="Installing the ODBC driver" style={{width: 
-'50%', 'background-color': 'transparent'}}/>
-<br/>
+### Power BI Installation
 
-When the driver installation is completed, you can verify the installation was successful:
-Search for ODBC in the Start menu and select "ODBC Data Sources **(64-bit)**".
+This tutorial assumes you have Microsoft Power BI Desktop installed on your Windows machine. You can download and install Power BI Desktop [here](https://www.microsoft.com/en-us/download/details.aspx?id=58494)
 
-<img src={require('./images/powerbi_02.png').default} class="image" alt="Creating a new ODBC Data Source"
-style={{width: '50%', 'background-color': 'transparent'}}/>
-<br/>
+We recommend updating to the latest Power BI version as the ClickHouse Connector is available in versions >= `2.137.751.0`
```

Comment:
Maybe we should add "available by default in versions >= `2.137.751.0`"? Otherwise, it might sound like it is only available for versions > `2.137.751.0`.

Reply:
"We recommend updating to the latest Power BI version as the ClickHouse Connector is available by default in versions >= `2.137.751.0`"

When I read it back, it doesn't feel nice grammatically. 

Suggestion: "We recommend updating to the latest version of Power BI. The ClickHouse Connector is available by default from version `2.137.751.0`."